src/SpectatorView.Unity/Assets/SpatialAlignment/Scripts/Utilities/Extensions.cs

```diff
@@ -1,237 +1,237 @@
-        /// bow out of the await that it can't cancel, but doesn't require completion/cancellation in order to cancel it's own execution.</remarks>
-        /// <param name="task">The task to await.</param>
-        /// <param name="cancellationToken">The cancellation token to stop awaiting.</param>
-        /// <returns>The task that can be awaited unless the cancellation token is triggered.</returns>
-        public async static Task<T> Unless<T>(this Task<T> task, CancellationToken cancellationToken)
-        {
-            return (await Task.WhenAny(task, cancellationToken.AsTask())) is Task<T> result ? result.Result : default(T);
-        }
-
-        /// <summary>
-        /// Helper class to enable await on <see cref="SynchronizationContext"/>. 
-        /// This is useful if you want to switch execution flow of an async function to a different thread, like Unity game thread for example.
-        /// </summary>
-        public struct SynchronizationContextAwaiter : INotifyCompletion
-        {
-            private static readonly SendOrPostCallback _postCallback = state => ((Action)state)();
-
-            private readonly SynchronizationContext _context;
-            public SynchronizationContextAwaiter(SynchronizationContext context)
-            {
-                _context = context;
-            }
-
-            public bool IsCompleted => _context == SynchronizationContext.Current;
-
-            public void OnCompleted(Action continuation)
-            {
-                _context.Post(_postCallback, continuation);
-            }
-
-            public void GetResult() { }
-        }
-
-        /// <summary>
-        /// Required extension method to enable awaiting on <see cref="SynchronizationContext"/>.
-        /// </summary>
-        /// <param name="context">Context to await (switch execution flow to).</param>
-        /// <returns>Awaiter for the "await" keyword to work.</returns>
-        public static SynchronizationContextAwaiter GetAwaiter(this SynchronizationContext context)
-        {
-            return new SynchronizationContextAwaiter(context);
-        }
-    }
-}
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using UQuaternion = UnityEngine.Quaternion;
+using UVector3 = UnityEngine.Vector3;
+
+namespace Microsoft.MixedReality.Experimental.SpatialAlignment.Common
+{
+    /// <summary>
+    /// Helper Unity side extensions to enable the core abstraction to be Unity agnostic.
+    /// </summary>
+    public static class Extensions
+    {
+        /// <summary>
+        /// Converts a <see cref="Vector3"/> to a <see cref="UVector3"/>.
+        /// </summary>
+        public static UVector3 AsUnityVector(this Vector3 input)
```

Comment:
>AsUnityVector [](start = 31, length = 13)

nit: i think andrei had hoped for these extension utilities to remain in the spatial alignment repo.